apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -375,6 +376,44 @@ API.v1.addRoute(
 	},
 );
 
+const getTeamByIdOrNameOrParentRoom = async (
+	params: { teamId: string } | { teamName: string } | { roomId: string },
+): Promise<Pick<ITeam, 'type' | 'roomId' | '_id'> | null> => {
+	if ('teamId' in params && params.teamId) {
+		return Team.getOneById<ITeam>(params.teamId, { projection: { type: 1, roomId: 1 } });
+	}
+	if ('teamName' in params && params.teamName) {
+		return Team.getOneByName(params.teamName, { projection: { type: 1, roomId: 1 } });
+	}
```

Comment:
is it required to support filter by name? otherwise I'd say to remove it

Reply:
I actually didn't add it at first. Matheus brought the point of all other team endpoints supporting both and i think he's got a point there, so...

Reply:
ic.. ok, better keep it consistent then